common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -655,6 +662,10 @@ div.problem {
       font-style: italic;
       -webkit-font-smoothing: antialiased;
     }
+
+    @media print {
```

Comment:
This presently hides the remaining checks information as well. It should be reasonably straightforward to be more specific to hide only buttons, but leave the check information.